package.json

```diff
@@ -21,14 +21,17 @@
   },
   "homepage": "https://github.com/northcoders/be-nc-news#readme",
   "devDependencies": {
+    "express": "^4.19.2",
     "husky": "^8.0.2",
     "jest": "^27.5.1",
     "jest-extended": "^2.0.0",
-    "pg-format": "^1.0.4"
+    "pg-format": "^1.0.4",
```

Comment:
pg-format should be a dependency rather than a dev dependency if it is being used in any of your model files